examples/basic/main.tf

```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+provider "datadog" {}
+
+module "datadog" {
+  #checkov:skip=CKV_TF_1: Irrelevant for this example
```

Comment:
```suggestion
```

Reply:
This is already excluded in the pipeline https://github.com/schubergphilis/terraform-aws-mcaf-datadog/blob/master/.github/workflows/terraform-validation.yaml#L157